### ISSUE TITLE
8$: suggest { } around arrow body on statement keyword

### ISSUE THREAD

**Issue by strager** [quick-lint/quick-lint-js#1097]
```javascript
let x = () => debugger;
// Intended: let x = () => {debugger;};
let y = () => while (true) {};
// Intended: let y = () => {while (true) {}};
```
We should detect `debugger`, `while`, `if`, `const`, and other things after `=>` and report a nice diagnostic.
